lib/immediate-juttle-job.js

```diff
@@ -43,6 +46,8 @@ class ImmediateJuttleJob extends JuttleJob {
             // We actually wait until close() to return the data.
         } else if (msg.type === 'error') {
             self._errors.push(msg.error);
+        } else if (msg.type === 'log') {
+            self._logs.push(msg);
```

Comment:
Actually I do not expect there to be a message of type `log` if `debug != true`. It's a moot point because the message is never forwarded to this function if debug is false.